arroyo/processing/strategies/streaming/transform.py

```diff
@@ -339,13 +365,14 @@ def __submit_batch(self) -> None:
         assert self.__batch_builder is not None
         batch = self.__batch_builder.build()
         logger.debug("Submitting %r to %r...", batch, self.__pool)
-        self.__results.append(
+        self.__processes.append(
```

Comment:
For the next time. Please consider keeping the cosmetic refactoring in a separate PR.
Reviewing would be easier if only the relevant changes were in the changeset.